nitrocli/src/arg_util.rs

```diff
@@ -25,6 +25,58 @@ macro_rules! count {
 }
 
 macro_rules! Command {
+  ( $name:ident, [ $( $var:ident($inner:ident) => ($str:expr, $exec:expr), ) *] ) => {
```

Comment:
Hm, is this a huge code duplication? But I can't seem to find it in the end result, so perhaps it's okay?

Reply:
(1) Only the `Display` implementation is duplicate – the other implementations are different (`Command::Variant` vs `Command::Variant(_)`). (2) The old macro variant is deleted after the migration.  In the first implementation, I used internal macros, but that just complicates the transition.